### PR TITLE
[PE-4870] Backstop test for Picker Component

### PR DESCRIPTION
**Pull request by sabah1910**
@jose @janibolkvadze @mattnickles https://ctl.atlassian.net/browse/PE-4870

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1036/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1037/ for more details.

**Comment by janibolkvadze** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://c57facda799ef280d13a9d2d02e0d75b58d45445-chi-pr.enterprise.ctl.io" target="_blank"> https://c57facda799ef280d13a9d2d02e0d75b58d45445-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1048/ for more details.

File: test/chi/components/picker.pug [hunk 1]
```diff
@@ -132,39 +132,40 @@ h4 With radio checked
...
+            span.chi-picker__radio
+            span.chi-picker__label(for='ri-Radio2') Radio2
+          .chi-picker__description
+            | Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer vehicula placerat iaculis. Ut rutrum pulvinar velit vitae molestie. Phasellus accumsan pharetra ex. Nulla iaculis velit sed velit vehicula dictum. Donec ut ultrices tortor. Vivamus sit amet lorem augue.
+
+h4 With description checked
+.test-with-description-checked
+  each type in ['Radio1','Radio2']
+    .chi-picker
+      input(type="radio", class="chi-picker__input", id=`id-checked-${type}`, checked=(type === 'Radio1'))
```
Comment (mattnickles) [inline test/chi/components/picker.pug:162]: Missing name attribute

```suggestion
      input(type='radio', class='chi-picker__input', name='with-radio-description-checked', id=`id-checked-${type}`, checked=(type === 'Radio1'))
```

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1058/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://24a6df431e9c7a0a0d98176462bb9c43d33a9884-chi-pr.enterprise.ctl.io" target="_blank"> https://24a6df431e9c7a0a0d98176462bb9c43d33a9884-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1059/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-20rhfl, https://ops.elasticbox.com/services/instances/i-fhvkvw

**Pull request merged**